saa_victoria_subtheme.theme

```diff
@@ -35,6 +35,9 @@ function saa_victoria_subtheme_preprocess_ds_entity_view(&$variables) {
     // get card variant info - if null, will use the 3rd value
     $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'card_type', 'su-card-wrapper--icon');
     $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'card_border_style', 'su-card-wrapper--border-regular');
+    $variables['attributes']['class'][] = $variables['content']['#paragraph']->getBehaviorSetting('react_paragraphs:card_variant_info', 'horizontal_card_image_style');
+    dpm($variables);
```

Comment:
The dpm shouldn't be here. :)

Reply:
oh woops! yes that wasn't supposed to be there haha.

Reply:
Let he or she who has never left a stray `dpm()` cast the first stone...

Reply:
No stones thrown. :) I've done the old `console.log()` before.